.github/workflows/aq_zombie_finder.yaml

```
@@ -0,0 +1,34 @@
+name: AQ Zombie Finder Unittest
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - "aq_zombie_finder/**"
+      - ".github/workflows/aq_zombie_finder.yaml"
+
+jobs:
+  test_and_lint:
```

```suggestion
  test_with_unittest:
```

You aren't running a linter here. So this is just to make it clearer
If you want to run a linter you can add this to the end of the file:
```
    - name: Analyse with pylint 
      run: |
          pylint --recursive yes test aq_zombie_finder
```
you'll also need to put a `.pylintrc` file in `aq_zombie_finder` folder - you can copy the one in the iriscasttools folder 

I recommend using a linter - it will basically analyse your code and give you potential ways to improve your code